### DISCUSSION TITLE
Reset Atom App

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#23700] [General]
Hello
I downloaded Atom which I like.
On the other hand, I downloaded too many packages and I would like to find the initial installation.
I uninstalled the app but every time I install it again I still have all the packages I want to get rid of.
Could someone help me please?
I'm using a MacBook Pro (13-inch, M1, 2020)
thanks in advance

**Comment by icecream17** [comment]
You can open in safe mode to launch without any packages

If you really want to completely reinstall: https://github.com/atom/atom/issues/23542 https://flight-manual.atom.io/faq/sections/how-do-i-uninstall-atom-on-macos/

**Reply by ghost** [reply]
Hello
Thank you for your help.
However, I have another question.
When I am on my index.html file in Atom how to execute my code? for example the "Run" function
thank you

**Reply by icecream17** [reply]
(in programming and many other fields, searching the web is usually much much faster than asking a question)
https://www.wikihow.com/Run-a-HTML-File

**Reply by vwallajabad** [reply]
You can use [atom-live-server](https://atom.io/packages/atom-live-server). This package will help you run your html code in local host.
